src/daemon/daemon.cpp

```diff
@@ -2134,6 +2140,15 @@ error_string mp::Daemon::async_wait_for_ssh_and_start_mounts_for(const std::stri
         auto vm = it->second;
         vm->wait_until_ssh_up(up_timeout);
 
+        if (server)
+        {
+            Reply reply;
+            reply.set_reply_message("Waiting for initialization to complete");
```

Comment:
I can't see this message in my testing. I confirm multipass waited for cloud init. Debugging, the code does go through here, but it does not cause the message to be shown in the client.

Reply:
Right, because it's nearly instantaneous for our default cloud-init.  In order to see it, you'll need a custom cloud-init that has a long running operation.

Reply:
I don't think that's it &ndash; I tried with a *sleep-30* cloud-init, and indeed the launch waits until the sleep is completed, but the message never shows.

Here is what I mean. The cloud init file:

```yaml
output: { all: "| tee -a /var/log/cloud-init-output.log" }
runcmd:
  - date
  - echo sleeping 30
  - sleep 30
  - echo slept
  - date
```
Then launching with `multipass launch --cloud-init ... && date`. Comparing timestamps you can see that the command indeed finishes only after cloud-init completes, but the msg does not show.

Reply:
Hmm, ok, I'll look into it.  Thanks!

Reply:
Found the issue and now fixed.

Reply:
Well, it prints this out now on start too.  I don't think we want that.

Reply:
Ok, that's better :wink: